src/test/java/com/google/sps/SyncServletTest.java

```diff
@@ -0,0 +1,119 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.when;
+import com.google.sps.data.*;
+import com.google.sps.servlets.SyncServlet;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Queue;
+import java.util.LinkedList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public final class SyncServletTest {
+    
+    private static final String UPDATE_STATE_PARAMETER = "userState";
+    private static final String VIDEO_TIMESTAMP_PARAMETER = "timeStamp";
+
+    @Test
+    public void testCorrectVideoStringOutput(){
+        Room testRoom = new Room(new LinkedList<Member>());
+        testRoom.addVideo(Video.createVideo("www.RossJohnson.com"));
+        String expected = "{\"currentState\":\"UNSTARTED\",\"currentVideoTimestamp\":0,\"url\":\"www.RossJohnson.com\"}";
+        
+        String actual = SyncServlet.roomToVideoJson(testRoom);
+        
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testVideoUpdateTimeStamp(){
+        long newTimestamp = 50;
```

Comment:
You can move all common testing variables as test class static constants.
 e.g 
public static long TestTimeStamp = 50; at top.